.github/workflows/new_contributor_pr.yml

```
@@ -13,11 +13,10 @@ jobs:
     name: Hello new contributor
     runs-on: ubuntu-latest
     steps:
-      # Pinned to v2.0
-      - uses: deborah-digges/new-pull-request-comment-action@224c179a9e23f65ec50ff3240b8716369dc415d7
+      - uses: actions/first-interaction@v1
         with:
-          access-token: ${{ secrets.GITHUB_TOKEN }}
-          message: |
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |
             Hello @{}! Thank you for your contribution 💪
```

[first-interaction](https://github.com/actions/first-interaction) doesn't mention any interpolation, so I think we need to stop trying to insert the username?

```suggestion
            Hello! Thank you for your contribution 💪
```

---

Good catch :eagle: